packages/web-components/src/components/ic-checkbox/ic-checkbox.tsx

```diff
@@ -183,7 +192,13 @@ export class Checkbox {
       : removeHiddenInput(this.host);
 
     return (
-      <Host class={{ ["disabled"]: this.disabled, ["small"]: this.small }}>
+      <Host
+        class={{
+          ["disabled"]: this.disabled,
+          ["small"]: this.small,
+          [`${this.size !== undefined ? this.size : parentElementSize}`]: true,
```

Comment:
```suggestion
          [`${this.size || parentElementSize}`]: true,
```

Might help readability